doc/nrf/introduction.rst

```diff
@@ -3,17 +3,51 @@
 About the |NCS|
 ###############
 
-The |NCS| is hosted, distributed, and supported by Nordic Semiconductor for product development with Nordic SoCs.
-The SDK is distributed in multiple Git repositories with application code, drivers, re-distributable libraries, and forks of open-source code used in the SDK.
+The nRF Connect SDK enables you to develop applications for nRF52, nRF53, and nRF91 Series devices.
+It is a set of open source projects maintained by Nordic Semiconductor, consisting of several repositories such as:
 
-The main repository is the ``nrf`` repository, which contains application and sample projects as well as the SDK manifest file that enables the repositories to be managed as one code base with the west tool.
+* `sdk-nrf`_ repository - contains applications, samples, libraries, and drivers that are specifically targeted at Nordic Semiconductor devices.
+* `sdk-nrfxlib`_ repository - contains closed-source libraries and modules in binary format.
+  See the :doc:`nrfxlib documentation <nrfxlib:README>`.
+* `sdk-mcuboot`_ repository - contains a fork of the MCUboot project, which provides a secure bootloader application.
+  You can find the fork in ``bootloader/mcuboot`` after obtaining the nRF Connect SDK source code.
+  See the :doc:`documentation <mcuboot:index>` in Nordic Semiconductor’s MCUboot fork.
+* `sdk-zephyr`_ repository - contains a fork of the `Zephyr`_ project, which provides samples, libraries, and drivers for a wide variety of devices, including Nordic Semiconductor devices.
+  See the :doc:`documentation <zephyr:index>` in Nordic Semiconductor’s Zephyr fork.
 
-See :ref:`getting_started` for information about how to install the |NCS| and about the first steps.
+The repositories contain the application code, drivers, re-distributable libraries, and forks of open-source code used in the SDK.
+
+The main repository is ``sdk-nrf``.
+It contains the SDK manifest file that enables you to manage the repositories as one code base with the ``west`` tool.
+
+Tools and configuration
+***********************
+
+The figure below visualizes the tools and configuration methods in |NCS|.
+They are based on the :ref:`Zephyr toolchain <zephyr:getting_started>`.
+All of them have a role in the creation of an application, from configuring the libraries or applications to building them.
+
+.. figure:: images/ncs-toolchain.png
```

Comment:
Don't think I can comment on the figure, but it says gcc instead of GCC, and Device Tree instead of devicetree.
Also, not sure how you created the figure, but if there is a (Visio) source file for the figure it should be included.

Reply:
Image stolen from DevZone blog and it is only in .png format. Maybe I can try to play with Visio and create a better one.